Kotlin/src/main/kotlin/com/gildedrose/Sulfuras.kt

```diff
@@ -0,0 +1,8 @@
+package com.gildedrose
+
+class Sulfuras() : ItemType("Sulfuras, Hand of Ragnaros", 0, 80, false) {
+    override fun updateQuality() {
+
+    }
```

Comment:
It sounds nitpicky but eliminating the empty line here will make it more readable and obvious that the body is intentionally empty, otherwise it may look like a mistake in the future.
```
    override fun updateQuality() {}
```

(and it just looks prettier!)